### DISCUSSION TITLE
Upgrade to ImageSharp v2 in v10

### DISCUSSION THREAD

**Discussion by arknu** [umbraco/Umbraco-CMS#11946] [Features and ideas]
Today ImageSharp v2 has been released, with a new ImageSharp.Web to follow shortly (https://sixlabors.com/posts/announcing-imagesharp-200/). As far I understand, there are a few breaking changes in the API, but nothing major.

Is ImageSharp v2 going to be adopted for v10? I very much hope so, as this version brings support for WebP, which has been sorely missing from Umbraco 9.

I didn't see any issue for this, hence this question. If there are not already plans, I might have a go once ImageSharp.Web v2 is out.

**Comment by abjerner** [comment]
There is a similar discussion: https://github.com/umbraco/Umbraco-CMS/discussions/11938

**Reply by arknu** [reply]
Ah, sorry about that. Searched issues, but not discussions... I will monitor the other discussion.

If possible, feel free to merge this discussion into the other one (it seems I can't do that).
